File: app/src/main/java/com/aero51/moviedatabase/ui/SettingsActivity.java
package com.aero51.moviedatabase.ui;

import android.app.ActionBar;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.aero51.moviedatabase.R;

import java.util.Objects;

public class SettingsActivity extends AppCompatActivity {
    public static final String
            KEY_PREF_EXAMPLE_SWITCH = "example_switch";
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);

        getSupportFragmentManager().beginTransaction()
                .replace(android.R.id.content, new SettingsFragment())
                .commit();


    }
}

File: app/src/main/java/com/aero51/moviedatabase/SettingsActivity.kt
package com.aero51.moviedatabase

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.aero51.moviedatabase.ui.SettingsFragment

class SettingsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                .replace(android.R.id.content, SettingsFragment())
                .commit()

        }
    }

    companion object {
        const val KEY_PREF_EXAMPLE_SWITCH = "example_switch"
    }
}